ItemList/Tests/ItemList.DependencyInjection.Tests/ContratsImplementationTests.cs

```diff
@@ -19,7 +19,65 @@ internal class ContratsImplementationTests
         [SetUp]
         public void SetUp()
         {
-            _containerMock = new ContainerMock();
+            _containerMock = new ContainerMock(new HashSet<string>());
+            SetUpResolverMock();
+        }
+
+        [Test]
+        public void AllContractsHaveRegisteredImplementation()
```

Comment:
The naming of tests should be consistent. In this case, something along the lines of `Bootstrap_KnownBootstrapper_RegistersAllContracts` would be a bit more fitting.